src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AppointmentController.kt

```diff
@@ -67,7 +67,7 @@ class AppointmentController(private val appointmentService: AppointmentService)
       )
     ]
   )
-  fun createAppointment(@RequestBody createAppointmentSpecification: CreateAppointmentSpecification): CreatedAppointmentDetailsDto =
+  fun createAppointment(@RequestBody createAppointmentSpecification: CreateAppointmentSpecification): List<CreatedAppointmentDetailsDto> =
```

Comment:
Hmm.. didn't really notice in last PR, but do we need to return this data? I suppose we have to be consistent with prison-api?!

Reply:
We don't need to really but felt like the correct thing to do.